src/app/core/header/header.component.ts

```diff
@@ -13,6 +13,11 @@ export class HeaderComponent implements OnInit {
 
   ngOnInit(): void {}
 
+  public get login(): string {
+    const { firstName, lastName } = this.authService.userInfo || {};
```

Comment:
I would get rid of the empty object.
If a user is logged in than he definitely has first and last names.